lib/internal/test_runner/test.js

```diff
@@ -655,6 +655,8 @@ class Test extends AsyncResource {
         this.reporter.coverage(this.nesting, kFilename, this.harness.coverage);
       }
 
+      this.reporter.endOfTests();
```

Comment:
use [`_flush`](https://nodejs.org/api/stream.html#transform_flushcallback) to spare this addition